Project.toml

```
@@ -18,10 +11,16 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[compat]
+DSP = "0.5.1, 0.6"
+FFTW = "0.2.4, 1.2"
```

```suggestion
FFTW = "0.3, 1"
```

`0.2.4` is a 0.7/1.0 version so we could drop it now.